routes/blog/[slug].tsx

```diff
@@ -0,0 +1,46 @@
+import { Handlers, PageProps } from "$fresh/server.ts";
+import { CSS, render } from "$gfm";
+import { Head } from "$fresh/runtime.ts";
+import { getPost, Post } from "@/utils/posts.ts";
+import Nav from "@/components/Nav.tsx";
+import Header from "@/components/Header.tsx";
+import { BlogHeaderNavItems } from "@/routes/blog/index.tsx";
+
+export const handler: Handlers<Post> = {
+  async GET(_req, ctx) {
+    const post = await getPost(ctx.params.slug);
+    if (post === null) return ctx.renderNotFound();
+    return ctx.render(post);
+  },
+};
+
+export default function PostPage(props: PageProps<Post>) {
+  const post = props.data;
+
+  const date = new Date(post.publishedAt).toLocaleDateString("en-us", {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+  });
+
+  return (
+    <>
+      <Head>
+        <style dangerouslySetInnerHTML={{ __html: CSS }} />
+      </Head>
+      <Header>
+        <Nav items={BlogHeaderNavItems} />
+      </Header>
+      <main class="max-w-screen-md px-4 pt-16 mx-auto">
```

Comment:
```suggestion
      <main class="max-w-screen-md px-4 py-16 mx-auto">
```
Otherwise, there's no bottom padding.

Reply:
Good catch! Thanks